hack/lib/init.sh

```diff
@@ -51,7 +51,7 @@ os::log::stacktrace::install
 os::util::environment::update_path_var
 
 if [[ -z "${OS_TMP_ENV_SET-}" ]]; then
-	os::util::environment::setup_tmpdir_vars "$( basename "$0" ".sh" )"
+	os::util::environment::setup_tmpdir_vars "$( basename -- "$0" ".sh" )"
```

Comment:
Maybe we should do:

```sh
if [[ "$0" ~= *.sh ]]; then
   os::util::environment::setup_tmpdir_vars "$( basename "$0" ".sh" )"
else
    os::util::environment::setup_tmpdir_vars "shell"
fi
```

The intent here was not to create a dir for the name of your shell

Reply:
update done.